### ISSUE TITLE
Crash on MacOS

### ISSUE THREAD

**Issue by Horneth** [rogchap/wombat#29]
Hi !

I've been trying to run the latest version on Mac 10.15.7 but it crashes immediately.
Only info I managed to get is this:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x477f9aa]

goroutine 46 [running]:
wombat/internal/app.(*api).GetWorkspaceOptions(0xc0004d8000, 0x4a1a4be, 0xb, 0xc0003701e0)
	/Users/runner/work/wombat/wombat/internal/app/api.go:174 +0x14a
wombat/internal/app.(*api).wailsReady(0xc0004d8000, 0x0, 0x0, 0x0)
	/Users/runner/work/wombat/wombat/internal/app/api.go:96 +0x157
created by github.com/wailsapp/wails/lib/event.(*Manager).Start.func1
	/Users/runner/go/pkg/mod/github.com/wailsapp/wails@v1.9.1/lib/event/manager.go:124 +0x457
```

Previous version runs flawlessly.
Let me know if there's something else I can do to give you more debug information.

**Comment by rogchap** [comment]
Thanks for reporting. When you said `Previous version runs flawlessly.` which version was that? If it was pre v0.2.0 then there is likely a database compatibility error.

Try removing all the files in `~/Library/Application Support/Wombat` and restarting. Does that work?

**Comment by rogchap** [comment]
Looking at the code I can still see a possible crash location. Thanks again. Will get a fix sorted.

**Comment by rogchap** [comment]
This should be fixed in v0.3.2 release: https://github.com/rogchap/wombat/releases/tag/v0.3.2

Let me know if these resolves it. Thanks again.

**Comment by Horneth** [comment]
It does work now, thanks !
